cmd/imagestream.go

```diff
@@ -33,12 +39,15 @@ func NewImageStreamCleanupCommand() *cobra.Command {
 		Run:     o.cleanupImageStreamTags,
 	}
 	cmd.Flags().BoolVarP(&o.Force, "force", "f", false, "delete image stream tags")
-	cmd.Flags().IntVarP(&o.CommitLimit, "git-commit-limit", "l", 100, "only look at the first <n> commits to compare with tags or use -1 for all commits")
+	cmd.Flags().IntVarP(&o.CommitLimit, "git-commit-limit", "l", 0, "only look at the first <n> commits to compare with tags or use 0 for all commits")
```

Comment:
(Technically changing the default value and all related changes have nothing to do with this PR)

Reply:
It was a requirement from the ticket that's why I included it, though you are right I could extract the change regarding the limit modification and add it into another commit